Dockerfile.build.template

```diff
@@ -0,0 +1,37 @@
+# Build nodejs dependencies
+FROM resin/%%ARCH%%-node:6.2-slim
```

Comment:
I added a comment. I'll update the version in a separate set of PRs. i'd like to get this moving